api/views/auth/social_auth_complete.py

```
@@ -0,0 +1,252 @@
+        except Exception as e:
+            logger.error(f"Facebook social login error: {str(e)}")
+            return Response(
+                {"error": "Facebook authentication failed", "detail": str(e)},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+
+class TelegramSocialLoginView(SocialLoginView):
+    """Telegram social login view using dj-rest-auth"""
+    adapter_class = TelegramProvider
+    permission_classes = [AllowAny]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)